### PR TITLE
[6] Manage repositories.

### PR DESCRIPTION
**Pull request by belov**
#6  Manage repositories.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,7 @@ yarn-debug.log*
 
 /persistent
 /coverage
+.env
+.generators
+.idea/
+.ruby-gemset
```
Comment (achempion) [inline .gitignore]: newline is missing

File: app/contexts/project_context.rb [hunk 2]
```diff
@@ -4,11 +4,55 @@
...
+    return work_dir.to_s if work_dir.exist? && work_dir.directory? && !work_dir.empty?
+
+    clone(project)
+  end
+
+  def clone(project)
+    repository = Octokit::Repository.from_url(project.internal_repository_url)
+    client = Octokit::Client.new
+    repository = client.repository repository
+    git = Git.clone(repository.ssh_url, repository.name, path: directory_for_clone)
```
Comment (achempion) [inline app/contexts/project_context.rb]: context is the abstraction to make database queries, they should not work with file structure

File: .rubocop.yml [hunk 3]
```diff
@@ -17,6 +18,8 @@ AllCops:
 Style/ModuleFunction:
   Enabled: false
 
+Metrics/LineLength:
```
Comment (achempion) [inline .rubocop.yml]: I think we should follow default value here

**Review by achempion — COMMENTED** [review]
Great work!

Please, add documentation for functions and types of accepted arguments (in YARD format).

File: .rubocop.yml (hunk 3 above)
Comment (belov) [reply .rubocop.yml]: Default and Standard - it`s good, but we are live in 2020 year, and we have 4k monitors. 80 symbols - reduce one-liners, variables, readability. Why suffer? 
I think, 120 symbols is compromise between 80 symbols and cancellations. :)

Comment (achempion) [reply .rubocop.yml]: I don't like all rubocop defaults as well (like argument alignment or method naming) but have decided to follow default rubocop as close as possible with this project.

I've changed all code generated by `rails new` to fit default config as well https://github.com/evrone/evrone_opensource/pull/2.

Comment (belov) [reply .rubocop.yml]: ok

File: .gitignore (hunk 1 above)
Comment (belov) [reply .gitignore]: fixed

**Pull request merged**